### PR TITLE
Raspberry Pi

### PR DESCRIPTION
**Pull request by kkaempf**
This adds instructions and examples for installing Elemental on Raspberry Pi.

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -19,13 +20,27 @@ with the only help of an Elemental Teal iso
...
+operating system layer. At its core, it only requires grub2, dracut, a kernel, and systemd.
 
 Its sole purpose is to run Kubernetes (k3s or RKE2), with everything controlled through Rancher Manager.
 
 Elemental Teal is built in the [openSUSE Build Service](https://build.opensuse.org/package/show/isv:Rancher:Elemental:Stable:Teal53/node-image)
 and available through the [openSUSE Registry](http://registry.opensuse.org/isv/rancher/elemental/stable/teal53/15.4/rancher/elemental-node-image/5.3:latest)
 
+#### Elemental on x86-64 hardware
+
+Elemental Teal is production ready and fully supported on x86-64 with Rancher 2.7.0.
```
Comment (mjura) [inline docs/quickstart.md:33]: Rancher v2.7.0

File: docs/quickstart.md [hunk 2]
```diff
@@ -19,13 +20,27 @@ with the only help of an Elemental Teal iso
...
 Elemental Teal is the combination of "SLE Micro for Rancher" with the Rancher Elemental stack
 
-SLE Micro for Rancher is a containerized and "stripped to the bones" operating system layer. It only requires grub2, dracut, a kernel, and systemd.
+SLE Micro for Rancher is a containerized and "stripped to the bones"
+operating system layer. At its core, it only requires grub2, dracut, a kernel, and systemd.
 
 Its sole purpose is to run Kubernetes (k3s or RKE2), with everything controlled through Rancher Manager.
 
 Elemental Teal is built in the [openSUSE Build Service](https://build.opensuse.org/package/show/isv:Rancher:Elemental:Stable:Teal53/node-image)
 and available through the [openSUSE Registry](http://registry.opensuse.org/isv/rancher/elemental/stable/teal53/15.4/rancher/elemental-node-image/5.3:latest)
```
Comment (fgiudici) [inline docs/quickstart.md:29]:
```suggestion
and available through the [openSUSE Registry](http://registry.opensuse.org/isv/rancher/elemental/stable/teal53/15.4/rancher/elemental-node-image/5.3:latest).
```

File: docs/quickstart.md [hunk 3]
```diff
@@ -19,13 +20,27 @@ with the only help of an Elemental Teal iso
 
 Elemental Teal is the combination of "SLE Micro for Rancher" with the Rancher Elemental stack
```
Comment (fgiudici) [inline docs/quickstart.md:21]:
```suggestion
Elemental Teal is the combination of "SLE Micro for Rancher" with the Rancher Elemental stack.
```

File: docs/quickstart.md [hunk 4]
```diff
@@ -54,10 +70,27 @@ cluster and taking care of creating inventories, registrations for machines and
 
 We will use the Helm package manager to install the elemental-operator chart into our cluster
```
Comment (fgiudici) [inline docs/quickstart.md:71]:
```suggestion
We will use the Helm package manager to install the elemental-operator chart into our cluster.
```

**Review by mjura — APPROVED** [review]
LGTM

File: docs/quickstart.md [hunk 5]
```diff
@@ -90,14 +123,30 @@ As you can see this is a very simple selector that checks the key `location` for
...
+</Tabs>
 
 This creates a `MachineRegistration` which will provide a unique URL which we will use with `elemental-register` to register
 the node during installation, so the operator can create a `MachineInventory` which will be using to bootstrap the node.
 See that we set the label that match our selector here already, although it can always be added later to the `MachineInventory`.
 
+
+
 :::warning warning
 Make sure to modify the registration.yaml above to set the proper install device to point to a valid device based on your node configuration(i.e. /dev/sda, /dev/vda, /dev/nvme0, etc...)
```
Comment (fgiudici) [inline docs/quickstart.md:147]:
```suggestion
Make sure to modify the registration.yaml above to set the proper install device to point to a valid device based on your node configuration (i.e. /dev/sda, /dev/vda, /dev/nvme0, etc...).
```

**Review by fgiudici — APPROVED** [review]
Nice, we have docs for RPi! 🎉 
PR looks good! 
I found some missing full stops at the end of some lines *not* touched by this PR. Just .
Feel free to ignore in this PR.

**Comment by kkaempf** [comment]
> Nice, we have docs for RPi! tada PR looks good! I found some missing full stops at the end of some lines _not_ touched by this PR. Feel free to ignore.

There are actually _many_ lines affected :wink:

**Pull request merged**